biaspaltsau_a/14.Ansible.Workshop/roles/redmine_all/tasks/main.yaml

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
wrong dividing of roles, you have all tasks in all roles, there are a lot of duplication code

Reply:
corrected